styleguide/STYLE.md

```diff
@@ -197,6 +197,15 @@ Internal links need to start with `/docs`, for example:
 Read more about [environment variables](/docs/pipelines/environment-variables)
 ```
 
+### Anchor links  
+
+To use an anchor link where you need to link to an H2-level heading, append the section's name to the main page link, for example:  
```

Comment:
Worth adding that these are generated automatically from the title, and viewable in the # that appears when you mouse over the heading?

Reply:
Yes, adding.